modules/zeotapIdPlusIdSystem.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This module adds Zeotap to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/zeotapIdPlusIdSystem
+ * @requires module:modules/userId
+ */
+import * as utils from '../src/utils.js'
+import {submodule} from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const ZEOTAP_COOKIE_NAME = 'IDP';
+const storage = getStorageManager();
+
+function isValidConfig(configParams) {
+  if (!configParams) {
+    utils.logError('User ID - zeotapId submodule requires configParams');
+    return false;
+  }
+  if (!configParams.name) {
+    utils.logError('User ID - zeotapId submodule requires valid config params: name');
+    return false;
+  }
+  return true;
+}
+
+function readCookie() {
+  return storage.cookiesAreEnabled ? storage.getCookie(ZEOTAP_COOKIE_NAME) : null;
+}
+
+function readFromLocalStorage() {
+  return storage.localStorageIsEnabled ? storage.getDataFromLocalStorage(ZEOTAP_COOKIE_NAME) : null;
+}
+
+function fetchId(configParams) {
+  if (!isValidConfig(configParams)) return undefined;
```

Comment:
per the above, i don't think you need this since you're not using any `configParams`
```suggestion
```

Reply:
made changes.